docs/guides/data-movement/copy-offload.md

```diff
@@ -71,13 +77,12 @@ The copy-offload container profile is specified in the `container` directive. Se
 #DW container name=copyoff-container profile=copy-offload-default [...]
 ```
 
-### Library libcopyoffload
+> [!NOTE]
+> See [User Containers](../user-containers/readme.md) for details about customizing the directives and the container profile for the storage resources created by the Workflow.
+
+### Use libcopyoffload
 
-Where to get it.
-Linking with it.
-The API.
-The `/etc/local-rabbit.conf` file that has the name of the matching rabbit.
-The NNF_CONTAINER_LAUNCHER environment variable.
+The [`libcopyoffload` library](https://github.com/NearNodeFlash/nnf-dm/tree/master/daemons/lib-copy-offload) must be linked into the user's compute application.
```

Comment:
Should the lib-copy-offload directory have a little readme that points back to this document? Perhaps it should also say look at the header file for documentation on how to use the library or something along those lines.

Reply:
https://github.com/NearNodeFlash/nnf-dm/pull/301